recipes/OpenMiChroM/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "OpenMiChroM" %}
+{% set version = "0.0.3" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/OpenMiChroM-{{ version }}.tar.gz
+  sha256: 835ed034621994bde1522760f4601b82721f9f7ad065c3caf1a9a165d15af2b8
```

Comment:
I think the checksum needs to be updated for the new version number...

Reply:
No worries, I can't count the number of times I've done that 😀

Reply:
Friday night here in Brazil, you know =]
I commit the change